### PR TITLE
fix: prettier update new rules applied

### PR DESCRIPTION
**Pull request by eddiejaoude**
## Fixes Issue

Prettier changed their rules, so applied it to the code base https://prettier.io/blog/2023/07/05/3.0.0.html#breaking-changes

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: pages/docs/environments/environment-variables.mdx [hunk 1]
```diff
@@ -92,3 +92,4 @@ export default ({ children }) => (
     {children}
   </DocsLayout>
 );
+```
```
Comment (dan-mba) [inline pages/docs/environments/environment-variables.mdx:95]: This is breaking the build

File: pages/docs/environments/debugging-in-vscode.mdx [hunk 2]
```diff
@@ -21,7 +23,8 @@ This documentation explains how you can debug your BioDrop frontend and backend
 
 1. If you don’t already have a `launch.json` file, create one in the `.vscode` directory at the root of the BioDrop project.
 2. Add the given JSON to the `.vscode/launch.json` file.
-<ClipboardCopy>
+   <ClipboardCopy>
```
Comment (dan-mba) [inline pages/docs/environments/debugging-in-vscode.mdx:26]: This is breaking the build

File: pages/docs/environments/environment-variables.mdx (hunk 1 above)
Comment (eddiejaoude) [reply pages/docs/environments/environment-variables.mdx:95]: thank you! not sure why that gets added each time 😢

File: pages/docs/environments/debugging-in-vscode.mdx (hunk 2 above)
Comment (eddiejaoude) [reply pages/docs/environments/debugging-in-vscode.mdx:26]: Ah thank you! 👍

File: pages/docs/environments/environment-variables.mdx (hunk 1 above)
Comment (dan-mba) [reply pages/docs/environments/environment-variables.mdx]: I think you might need to add this file to `.prettierignore`.

The support for MDX has some issues, including triple backticks.

**Pull request merged**